### PR TITLE
Feature 109 print styles

### LINKED ISSUE #109: Print styles

### PR DESCRIPTION
**Pull request by platonovsimeon**
Resolves #109

### CODE REVIEW

File: dangerous.php [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```
Comment (anttileppa) [inline dangerous.php:5]: If 8419 is a customer number or something like that, it should be in the settings

File: fullscreen.php [hunk 2]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```
Comment (anttileppa) [inline fullscreen.php:5]: Instead of repeating the script to all pages, could this be moved into some generic template file?

File: index.php [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <html lang="fi">
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```
Comment (anttileppa) [inline index.php:4]: See above

File: smallgutter.php [hunk 4]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <?php require get_template_directory() . '/inc/template-head-generic.php'; ?>
+    <script src="//cdn1.readspeaker.com/script/8419/webReader/webReader.js?pids=wr" type="text/javascript"></script>
```
Comment (anttileppa) [inline smallgutter.php:5]: See above

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: smallgutter.php (hunk 4 above)
Comment (platonovsimeon) [reply smallgutter.php:5]: Fixed

File: index.php (hunk 3 above)
Comment (platonovsimeon) [reply index.php]: Fixed

File: fullscreen.php (hunk 2 above)
Comment (platonovsimeon) [reply fullscreen.php]: Fixed

File: dangerous.php (hunk 1 above)
Comment (platonovsimeon) [reply dangerous.php]: Fixed

**Pull request merged**